package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "core-js": "^3.6.5",
-    "fondue": "git+https://github.com/Wakamai-Fondue/wakamai-fondue-engine.git#a9c61d236e2430dc0d594665f069ab8e59269065",
+    "@wakamai-fondue/engine": "git+https://github.com/Wakamai-Fondue/wakamai-fondue-engine.git#eeb75565757d5243803adb129b46f38ac997e92d",
```

Comment:
Apologies for asking something you likely already explained to me, but is it possible to point to just the most recent `master` version of engine here? This seems like something that requires not forgetting to update it all the time ;-)

If this is because of an unreliability in yarn, can we address that on their side or move to another package manager?

Reply:
Both yarn and npm resolve `master` to a specific commit, which is locked in `yarn.lock` or `package-lock.json`.
In yarn it works fine to specify `master` in package.json and run `yarn update <lib>` to update the reference to the latest commit. In NPM this doesn't work well, NPM doesn't seem to check the git repo for updates, which is why we explicitly reference the commit in `package.json` here.